schema/types/suricata.schema

```diff
@@ -76,10 +76,10 @@ type suricata.alert = suricata.component.common + record {
     }
   },
   flow: suricata.component.flow,
-  payload: string,
+  payload: blob,
   payload_printable: string,
   stream: uint64,
-  packet: string,
+  packet: blob,
```

Comment:
I probably missed a bunch of these, but I wanted to test that it works. Also, do we actually already want to change the schemas or do we wait with that? It can break queries such as `where "abc" in payload`.

Reply:
My hunch: Yes, just make the change.

@mavam can you confirm?

Reply:
Confirming, I'm not aware of anyone who does `"abc" in payload` on binary data.

Reply:
Then let's keep it. If I missed any `string` fields that should really be `blob`, we can update it as we go.